eng/testing/linker/SupportFiles/Directory.Build.targets

```diff
@@ -48,4 +62,9 @@
       TargetFramework="$(TargetFramework)" />
   </ItemGroup>
 
+  <PropertyGroup>
+    <BundleDir>$([MSBuild]::NormalizeDirectory('$(MSBuildProjectDirectory)', '$(OutputPath)', 'AppBundle'))</BundleDir>
```

Comment:
(nit) is there a reason these properties are at the bottom of the file? Usually they go at the top.